lib/autoload/kuroko2/command/shell.rb

```diff
@@ -50,13 +50,24 @@ def invoke(execution)
         command = execution.shell
         env     = execution.context.fetch('ENV', {})
 
+        worker_log = Kuroko2::WorkerLog.create(
```

Comment:
I worry that a lot of records are made.
You should add a script that cleaning old records like https://github.com/cookpad/kuroko2/blob/b21749d9e9e65211db8656b5fd719e0ec2c601d8/bin/cleanup_old_instances.rb

Reply:
I added it in https://github.com/cookpad/kuroko2/pull/98/commits/7cd2bbdf81e61e82c661d2526055b85f69aadbc3.